UnitTestBot/UTBotJava
# USVM to UtBot adapters without contest hacks

## Description

1. Adapters from USVM output to UtBot codegen input are added
2. New `Tool.USVM` is introduced in `ContestEstimator` (set `val mainTool = Tool.USVM` to use it)

## How to test

### Automated tests

Regression tests should pass, nightly statistics monitoring should continue working.

### Manual tests

1. Use Java 17
2. Configure `val mainTool = Tool.USVM` in `ContestEstimator`
3. Run `:utbot-junit-contest:run` task (some USVM warnings/errors can get logged)
4. There should be tests generated by USVM in `UTBotJava/utbot-junit-contest/build/output/test`

## Self-check list
